python/GafferSceneUI/DeletePointsUI.py

```diff
@@ -58,9 +72,40 @@
 			"description",
 			"""
 			Vertex interpolated int, float or bool primitive variable to choose which points to delete. Note a non-zero value indicates the point will be deleted.
+			""",
+
+			"layout:visibilityActivator", lambda plug : plug.node()["selectionMode"].getValue() == GafferScene.DeletePoints.SelectionMode.VertexPrimitiveVariable
+		],
+
+		"idListVariable" : [
```

Comment:
@murraystevenson, would be interested in your thoughts on this. Including "don't overthink it Haddon" if that's the case.

Reply:
I think there's a typo here - you meant "all the other examples I can think of are referring to *context* variables", right?

As an alternative, what if I explicitly spelled out "idListPrimitiveVariable"? It would remove any confusion, but I guess there's still a consistency problem.

To me, a bare idList referring to the name of the primitive variable to find the value in feels a bit underspecified, but I guess it probably is consistent - there are a bunch of other existing places that feel a bit underspecified to me in exactly the same way.

If we want to use explicitIdList for a plug that is actually an id list rather than the name of a primvar, I guess I don't have any real problem with that, though I would be interested in hearing Murray's take.

Reply:
Chatted to Murray about this, and we decided the "ID List Primitive Variable" label on the `selectionMode` menu provides enough context that we shouldn't worry about the ambiguity.